app/models/review.rb

```diff
@@ -0,0 +1,11 @@
+class Review < ActiveRecord::Base
+  belongs_to :food_truck
+  belongs_to :user
+
+  validates :header, presence: true
+  validates :body, presence: true
+  validates :rating, presence: true, numericality: { only_integer: true }
+  validates_inclusion_of :rating, :in => 1..5
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer the new style validations `validates :column, inclusion: value` over `validates_inclusion_of`.